rails/README.md

```diff
@@ -3,6 +3,8 @@
 - Use private instead of protected when defining controller methods.
 - Name date columns with `_on` suffixes.
 - Name datetime columns with `_at` suffixes.
+- Back boolean concepts like deleted?, published?, or active? with timestamps
+  via [Time for A Boolean](https://github.com/calebhearth/time_for_a_boolean)
```

Comment:
I think it's worth adding a _why_. I think I've heard we're doing an effort on explaining some of the reasoning in the guides.